cmd/minishift/cmd/openshift/service.go

```diff
@@ -19,39 +19,27 @@ package openshift
 import (
 	"fmt"
 	"os"
-	"strings"
-	"text/template"
 
 	"github.com/docker/machine/libmachine"
 	"github.com/pkg/browser"
 	"github.com/spf13/cobra"
 
-	"github.com/minishift/minishift/pkg/minikube/cluster"
 	"github.com/minishift/minishift/pkg/minikube/constants"
 	"github.com/minishift/minishift/pkg/util/os/atexit"
+
+	"github.com/minishift/minishift/pkg/minishift/openshift"
 )
 
 var (
-	namespace          string
-	serviceURLMode     bool
-	serviceURLFormat   string
-	serviceURLTemplate *template.Template
-	https              bool
+	namespace      string
+	serviceURLMode bool
```

Comment:
I meant the naming inside the code is meaningless (it is not clear from the name what the intent/usage is). using the `--url` flag forces `--urlmode`... which means, the display result in console?

http://www.itiseezee.com/?p=83